<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDlg</class>
 <widget class="QDialog" name="SettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="1">
      <widget class="QComboBox" name="ArrRegion">
       <property name="toolTip">
        <string>The region where the remote rendering session should be started. Choose the region that is closest to your physical location.</string>
       </property>
       <property name="accessibleName">
        <string>ARR Rendering Region</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Account Domain:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Blob Endpoint:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="TestStorage">
         <property name="toolTip">
          <string>Tests whether the connection to the storage account works.</string>
         </property>
         <property name="text">
          <string>Test Storage Account Settings</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ArrAccountKey">
       <property name="toolTip">
        <string>The key for the remote rendering account.</string>
       </property>
       <property name="accessibleName">
        <string>ARR Account Key</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="TestArr">
         <property name="toolTip">
          <string>Tests whether the connection to the ARR account works.</string>
         </property>
         <property name="text">
          <string>Test ARR Account Settings</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="ArrAccountDomain">
       <property name="toolTip">
        <string>Where the remote rendering account was set up.</string>
       </property>
       <property name="accessibleName">
        <string>ARR Account Domain</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Connection Profile:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Key:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Account Key:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="StorageKey">
       <property name="toolTip">
        <string>The key to connect to the storage account.</string>
       </property>
       <property name="accessibleName">
        <string>Storage Account Key</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="StorageEndpoint">
       <property name="toolTip">
        <string>The endpoint URL for the storage account. For example: https://&lt;name&gt;.blob.core.windows.net</string>
       </property>
       <property name="accessibleName">
        <string>Storage Blob Endpoint</string>
       </property>
       <property name="placeholderText">
        <string>For example: https://&lt;name&gt;.blob.core.windows.net</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ARR Account:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Storage Account:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Region:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="ProfileDropDown">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>List of connection profiles to switch between accounts.</string>
         </property>
         <property name="accessibleName">
          <string>Connection Profile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="AddProfile">
         <property name="toolTip">
          <string>Adds a connection profile.</string>
         </property>
         <property name="accessibleName">
          <string>Add Connection Profile</string>
         </property>
         <property name="text">
          <string> + </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="RemoveProfile">
         <property name="toolTip">
          <string>Removes the currently selected connection profile.</string>
         </property>
         <property name="accessibleName">
          <string>Remove Connection Profile</string>
         </property>
         <property name="text">
          <string> X </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ArrAccountID">
       <property name="toolTip">
        <string>The ID for the remote rendering account.</string>
       </property>
       <property name="accessibleName">
        <string>ARR Account ID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Account ID:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="StorageName">
       <property name="toolTip">
        <string>The name of the storage account to connect to.</string>
       </property>
       <property name="accessibleName">
        <string>Storage Account Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>66</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="Buttons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ProfileDropDown</tabstop>
  <tabstop>AddProfile</tabstop>
  <tabstop>RemoveProfile</tabstop>
  <tabstop>ArrAccountID</tabstop>
  <tabstop>ArrAccountKey</tabstop>
  <tabstop>ArrAccountDomain</tabstop>
  <tabstop>ArrRegion</tabstop>
  <tabstop>TestArr</tabstop>
  <tabstop>StorageName</tabstop>
  <tabstop>StorageKey</tabstop>
  <tabstop>StorageEndpoint</tabstop>
  <tabstop>TestStorage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
